draft-ietf-tls-esni.md

```diff
@@ -1112,7 +1112,7 @@ loses its ECH keys, or if a deployment of ECH must be rolled back on the server.
 
 The retry mechanism repairs inconsistencies, provided the server is
 authoritative for the public name. If server and advertised keys mismatch, the
-server will respond with ech_retry_requested. If the server does not understand
+server will respond with "retry_configs". If the server does not understand
```

Comment:
Nice catch! Perhaps:

> the server will reject ECH and respond with "retry_configs"

(I think, at the time, ech_retry_requested was part of the accept/reject signal.)

Reply:
Either fix is fine by me.

Reply:
```suggestion
server will reject ECH and respond with "retry_configs". If the server does
not understand
```